e2e-tests/content-strings/fields/insurance/policy/index.js

```diff
@@ -276,4 +277,18 @@ export const POLICY_FIELDS = {
       },
     },
   },
+  BANK_DETAILS: {
```

Comment:
Can we call this and all respective directories, files, ID etc, `LOSS_PAYEE_BANK_DETAILS`, this is specific to a loss payee.

I'm wondering if we should call this and all respective directories, files etc, `LOSS_PAYEE_UK_BANK_DETAILS` so it's clear, otherwise we have a "hidden assumption" that `X_BANK_DETAILS` is UK - wouldn't be obvious for a new developer. Thoughts?